src/components/CreatePlanModal/ModalSteps/PlanName.js

```diff
@@ -1,33 +1,51 @@
-import React from 'react';
+import React, { Component } from 'react';
 
 import {
     Form,
     FormGroup,
     TextInput
 } from '@patternfly/react-core';
 
-const PlanName = () => {
-
-    return (
-        <React.Fragment>
-            <h2> Name your plan </h2>
-            <Form>
-                <FormGroup
-                    label="Plan Name"
-                    isRequired
-                    fieldId="plan-name"
-                >
-                    <TextInput
+class PlanName extends Component {
+
+    constructor () {
+        super();
+        this.state = {
+            value: ''
+        };
+        this.handleTextInputChange = this.handleTextInputChange.bind(this);
```

Comment:
@ryelo wait, isn't the only reason why you are using class properties below to avoid having to do this binding?

Reply:
Yep! Totally missed that.